Client/Frontend/Strings.swift

```diff
@@ -56,7 +56,28 @@ private func MZLocalizedString(
 // MARK: - Alerts
 extension String {
     public struct Alerts {
-
+        public struct RestoreTabs {
+            public static let Title = MZLocalizedString(
+                "Alerts.RestoreTabs.Title.v109",
+                tableName: "Alerts",
+                value: "Firefox crashed. Restore your tabs?",
+                comment: "The title of the restore tabs pop-up alert.")
```

Comment:
I never know, do we need to give context to translators as to what us restore tabs, and why restore tabs alert would show?

Reply:
I'm not sure either, honestly.

@Delphine any suggestions?

Reply:
So this specific string seems pretty straight forward, since the string actually explains the comment.
But the following ones that refer to "restore tabs" seem less straight forward without more context (keep in mind it's not necessarily the same localizer working on all these strings, so they could lose context).
I'll comment on the other strings to give some possible ideas :)
More documentation on comments for l10n [here](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#add-localization-notes), in case that helps